src/pages/styles.less

```diff
@@ -140,12 +138,40 @@
     @apply shadow-none;
   }
 
-  .transparent-navbar, .transition-navbar:hover {
+  .transparent-navbar,
+  .transition-navbar:hover {
     @apply text-white;
   }
 }
 
-.card-container {
+.card-custom-border {
   width: 250px;
   height: 250px;
-}
\ No newline at end of file
+  background: rgba(221, 229, 239, 0.35);
+  border-radius: 50px 0px;
+}
+
+.custom-border {
+  border-radius: 30px 0px 30px 30px;
+  background: #dde5ef;
+  height: 210px;
+}
+
+.img-container {
+  border-radius: 50px;
+  height: 40px;
+  width: 40px;
+  background: red;
+  position: relative;
+}
+
+.rotate {
```

Comment:
Looks like this rotate class isn't getting used anywhere?

Reply:
That's true, it has been removed.